composer.json

```diff
@@ -3,22 +3,22 @@
     "description": "Migrate project",
     "license": "MIT",
     "require": {
-        "PHP": "^7.4",
+        "php": "^8.4",
         "keboola/encryption-api-php-client": "^0.2.2",
-        "keboola/job-queue-api-php-client": "^0.4.1",
-        "keboola/php-component": "^8.3",
-        "keboola/storage-api-client": "^12.2",
-        "keboola/sync-actions-client": "1.0",
-        "keboola/syrup-php-client": "^5.1"
+        "keboola/job-queue-api-php-client": "^5.2",
+        "keboola/php-component": "^11.0",
+        "keboola/storage-api-client": "^18.4",
+        "keboola/sync-actions-client": "^3.0",
+        "keboola/syrup-php-client": "^6.0"
```

Comment:
I think syrup client is no longer needed, so it should be removed from dependencies same as orphaned class imports